### DISCUSSION TITLE
Adwaita Blacklisting in Cinnamon

### DISCUSSION THREAD

**Discussion by Enokilis** [linuxmint/discussions#468] [Development]
The main repo recently had commits to blacklist the Adwaita theme due to missing icons.
This may not be necessary now that GNOME has released a legacy theme, which supposedly acts as an inheritee to Adwaita.
https://discourse.gnome.org/t/adwaita-icon-theme-legacy-release/21021

Would it be worth reverting the decision if this turns out to be a viable solution?

**Comment by eugenialoli** [comment]
Yeah, I think it should be reverted. Adwaita should be there if it has all the icons necessary.

**Comment by Enokilis** [comment]
Arch just released the legacy package, so this can readily be deployed and tried. It turns out that the canonical Adwaita theme is still problematic, as it leads to a mixed theme based on conditional inheritance.
It's an infinitely better situation than having missing icons, but it's arguably still worth addressing since it's the default theme for many setups, including the configuration that upstream Cinnamon ships with.

If Adwaita were turned into an alias of AdwaitaLegacy, everything should work out and leave no users forced to update their theme config.
